src/pudl/transform/params/ferc1.py

```diff
@@ -3980,7 +3980,6 @@
             # Note: this table does not currently get exploded. It will require
             # additional debugging at a later date.
             "calculation_tolerance": 0.4,
-            "subtotal_column": "depreciation_type",
```

Comment:
this subtotal column is also the `xbrl_factoid` column. i think it was here bc there are `"total"`s in this column? @e-belfer i removed it bc it was somewhat duplicative to that the factoid and the subtotal be the same - it was junking up the idea of the primary key for the `table_dimensions_ferc1`

bc this table isn't being slurped into the explosion it felt okay to just ✂️  it

Reply:
Yes that makes sense to me.